README.md

```diff
@@ -97,6 +97,28 @@ We're actively working on simple installation steps for other platforms. For now
 
 ### Build system
 
+#### Windows
```

Comment:
I'd probably put Windows below Linux and Mac here and elsewhere.

Reply:
done.